src/arc_borrow.rs

```diff
@@ -46,6 +46,9 @@ impl<'a, T> ArcBorrow<'a, T> {
     /// e.g. if we obtain such a reference over FFI
     /// TODO: should from_ref be relaxed to unsized types? It can't be
     /// converted back to an Arc right now for unsized types.
+    ///
+    ///  # Safety
+    /// - The reference to `T` must be valid
```

Comment:
must be a reference to a T that came from an Arc.

"Reference must be valid" is not meaningful, it is already UB to have an invalid reference

Reply:
Ah, true! Do you think this function needs a safety section then?

Reply:
Or, if calling it with an invalid reference is already UB, and we are not doing anything that can cause UB, does this function need to be unsafe?

Reply:
Yes, this function needs a safety section, because calling it with an arbitrary &T can cause unsoundness down the line. It must be called with an &T that came from an Arc

Reply:
Ok, thank you. Is like a `from_raw`?

Reply:
Which `from_raw`?

Reply:
I meant the one in std::sync::Arc, which seems like a similar function?

Reply:
Yes. In this case it is that this reference must have been obtained from a triomphe Arc, UniqueArc, or ArcBorrow.

Reply:
Great, makes sense!

Reply:
why remove it?

Reply:
Is this better:
```
# Safety
- The reference to `T` must have come from a Triomphe Arc, UniqueArc, or ArcBorrow.
```

Reply:
yes